src_new/adapters/prebid.js

```diff
@@ -591,6 +595,13 @@ function fetchBids(activeSlots, impressionID){
 					};
 
 				}
+				if(partnerToEnableSRA && partnerToEnableSRA.length>0){
```

Comment:
no need to maintain partnerToEnableSRA,
no need to loop
Move this logic to line 578, use CONFIG.forEachAdapter,  just check in constants if the current adapter is in SRA_ENABLED_BIDDERS, if yes then add the config, you can move this code in a separate function

Reply:
Do we need a new function to generate config ? or just move prebidConfig variable declaration above and add singleRequest in that prebidConfig object ?

Reply:
Yeah 
Updated the code 

Added a new function